### PR TITLE
update SQL Server README

### PR DESCRIPTION
**Pull request by pranshi06**
### What

1. Fix typo
2. Add extra instructions (for webpage rendering, since it does not show the title of the bash script)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,7 +136,7 @@ HASURA_DDN_PAT=$(ddn auth print-pat) docker compose -f docker-compose.hasura.yam
 
 Finally, now that our `DataConnectorLink` has the correct environment variables configured for the SQL Server connector,
 we can run the update command to have the CLI look at the configuration JSON and transform it to reflect our database's
-schema in `hml` format. In a new terminal tab, run:
+schema in `hml` format. In a new terminal tab from the root of your project, run:
```
Comment (rsd1122) [inline README.md:139]:
```suggestion
schema in `hml` format. In a new terminal tab from your project's root directory run:
```

File: README.md [hunk 2]
```diff
@@ -143,7 +147,7 @@ scaffolded out for you 🎉
 
 ### 8. Import _all_ your indices
 
-You can do this in one convenience command.
+You can do this in one convenience command. From the root of your project, run:
```
Comment (rsd1122) [inline README.md:150]:
```suggestion
You can do this with just one command. From your project's root directory, run:
```

File: README.md [hunk 3]
```diff
@@ -152,7 +156,7 @@ ddn connector-link update my_sql --subgraph my_subgraph --add-all-resources
 ### 9. Create a supergraph build
 
 Pass the `local` subcommand along with specifying the output directory as `./engine` in the root of the project. This
-directory is used by the docker-compose file to serve the engine locally:
+directory is used by the docker-compose file to serve the engine locally. From the root of your project, run:
```
Comment (rsd1122) [inline README.md:159]:
```suggestion
directory is used by the docker-compose file to serve the engine locally. From your project's root directory, run:
```

**Review by rsd1122 — CHANGES_REQUESTED** [review]

**Comment by pranshi06** [comment]
@rsd1122 I've addressed the review comments. Please have a look.

**Review by rsd1122 — APPROVED** [review]

**Comment by rsd1122** [comment]
Changes approved. Feel free to merge.

**Review by rsd1122 — APPROVED** [review]

**Review by codingkarthik — APPROVED** [review]

**Pull request merged**